src/ui/public/kbn_top_nav/bread_crumbs/bread_crumb_urls.js

```diff
@@ -14,7 +14,7 @@
  */
 export function getBreadCrumbUrls(breadcrumbs, url) {
   return breadcrumbs.map(breadcrumb => {
-    const breadCrumbStartIndex = url.toLowerCase().lastIndexOf(breadcrumb.toLowerCase());
+    const breadCrumbStartIndex = url.toLowerCase().lastIndexOf(breadcrumb.replace(/\W/, '').toLowerCase());
```

Comment:
@ycombinator actually for very good reason. because of this line: https://github.com/elastic/kibana/blob/master/src/ui/public/kbn_top_nav/bread_crumbs/bread_crumbs.js#L27

The path "savedVisualizations" becomes "Saved Visualizations" because of the `_.startCase` call. However this string is also used to build the breadcrumb to match back to the path, and since a space is inserted it is unable to match back to the correct directory. I could just remove all spaces if that would be preferred.